### ISSUE TITLE
NOT FIXED: How to restore Windows 11 desktop icons and behavior after uninstall #2251

### ISSUE THREAD

**Issue by pshellooe** [valinet/ExplorerPatcher#2253]
Thank you for the quick response. The issue is not the grid. I tried your suggestion and the look-and-feel is the same.

The issue is that my Windows 11 desktop is now behaving like Windows 10 (or even worse XP - Win 8). See the attached videos of the before and after behavior. The new Windows 11 desktop layout and behavior is a big improvement, I don't know why anyone would want to revert back to what it was before. 

BEFORE
[https://www.youtube.com/shorts/iSW-A_PUwK8](url)

AFTER
[https://www.youtube.com/shorts/I_ACIPy1xGM](url)

**Comment by Amrsatrio** [comment]
If you want your desktop to behave like "AFTER", then check this menu item:

<img width="442" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/20662640/4666567e-8681-486b-87b3-74b679e3129f">

Dude... how long have you been using Windows? I'll leave the resolution of this issue to you.

**Comment by Amrsatrio** [comment]
The same toggle in the new right click menu:

<img width="424" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/20662640/5e9a97fc-0191-4c5c-b2c7-691c9aa2f660">

**Comment by pshellooe** [comment]
I've been using Windows since it was introduced (yeah, I'm old). Your suggestion does not fix the issue. Fortunately, the PC that I tried ExplorerPatcher on is a new build. I am just going to do a system restore to fix the issue. I thought that some of the tweaks would be interesting (making the Taskbar smaller, etc.), but I just discovered that Microsoft is doing the right thing with the UI improvements from 10 to 11. Maybe I'm alone in thinking that, but going back to how things used to be just isn't for me.
